knative/knative-eventing-install/base/cluster-role-binding.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
 metadata:
   labels:
-    eventing.knative.dev/release: "v0.11.0"
+    eventing.knative.dev/release: v0.14.2
```

Comment:
Should this also be 0.14.3?

Reply:
0.14.2 is the actual version of the release I used since there wasn't a 0.14.3 releases for knative-eventing.